src/pages/Landing/AboutUs.jsx

```diff
@@ -0,0 +1,124 @@
+import icon08 from 'svg/veja_web_pag_landingpage_Icons-08.svg';
+import icon09 from 'svg/veja_web_pag_landingpage_Icons-09.svg';
+import icon10 from 'svg/veja_web_pag_landingpage_Icons-10.svg';
+import icon11 from 'svg/veja_web_pag_landingpage_Icons-11.svg';
+import icon12 from 'svg/veja_web_pag_landingpage_Icons-12.svg';
+import icon13 from 'svg/veja_web_pag_landingpage_Icons-13.svg';
+import icon14 from 'svg/veja_web_pag_landingpage_Icons-14.svg';
+
+const AboutUs = () => {
+    return (
+        <div className={styles.bgContainer}>
+
+            <div className={styles.mdContainer}>
+                <div className={styles.titles}>
+                    <h3 className={styles.title}>Sobre nós</h3>
+                    <p className={styles.subtitle}>Sabe mais sobre o nosso projecto.</p>
+                </div>
+                <p className={styles.text}>
+                    Vejo em Casa é uma plataforma sem fins lucrativos de conteúdos diversificados em formato live streaming, de cariz solidário e desenvolvida por uma equipa de voluntários. Este projeto integra o movimento tech4COVID19, cujo propósito é a criação de soluções que atenuem as consequências desta pandemia.
+                    <br /><br />
+                    Sentimos que este é o momento ideal para dar a volta por cima, para conhecer talentos e conteúdos do melhor que há, para partilhar e, acima de tudo, para ajudar. Por isso, todos os nossos eventos oferecem a oportunidade ao público de apoiar diretamente não só os criadores, mas também o movimento Tech4COVID19 e as suas iniciativas.
+                </p>
+                <div className={styles.powered}>Powered by:</div>
+                <img src={logo} alt='Logotipo do #tech4COVID19' className={styles.logo} />
+            </div>
+
+            <div className={styles.smContainer}>
+                <div className={styles.icons}>
+                    <div>
+                        <img 
+                            src={icon06} 
+                            alt='icon que simboliza tecnologia' 
+                            className={styles.icon}
+                            style={{width: '78px'}}
+                        />
+                        <p style={{color: '#FD641E'}}>TECNOLOGIA</p>
+                    </div>
+                    <div>
+                        <img 
+                            src={icon07} 
+                            alt='icon que simboliza música' 
+                            className={styles.icon}
+                            style={{width: '78px'}}
+                        />
+                        <p style={{color: '#EB2907'}}>MÚSICA</p>
+                    </div>
+                    <div>
+                        <img 
+                            src={icon08} 
+                            alt='icon que simboliza arte e design' 
+                            className={styles.icon}
+                            style={{width: '67px'}}
```

Comment:
It would be nice if we keep all the CSS in the same file. But I see you point about the icons. They all have different sizes. Maybe we can talk with the designer to draw a box with the same dimensions around the icons which will help us on the front-end. But for now it works 👍

Reply:
I can put it in the css file.
Having the icons with the same dimension would definitely be better!